website/docs/configuration/index.html.md

```diff
@@ -16,23 +16,52 @@ describing an intended goal rather than the steps to reach that goal.
 
 ## Resources and Modules
 
-The main purpose of the Terraform language is declaring [resources](/docs/configuration/resources.html).
+The main purpose of the Terraform language is declaring [resources](./resources.html).
 All other language features exist only to make the definition of resources
 more flexible and convenient.
 
-A group of resources can be gathered into a [module](/docs/configuration/modules.html),
+A group of resources can be gathered into a [module](./modules.html),
 which creates a larger unit of configuration. A resource describes a single
 infrastructure object, while a module might describe a set of objects and the
 necessary relationships between them in order to create a higher-level system.
 
 A _Terraform configuration_ consists of a _root module_, where evaluation
-begins, along with a tree of child modules created when one module references
+begins, along with a tree of child modules created when one module calls
 another.
 
+## Arguments, Blocks, and Expressions
```

Comment:
This is the bit where I was talking about adding a capsule version of the syntax outline to index.html, so that readers can stay away from syntax.html if they want.